sdk/ai/ai-inference-rest/samples/v1-beta/javascript/imageEmbeddings.js

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/**
+ * Demonstrates how to get image embeddings from a model endpoint.
+ *
+ * @summary Get image embeddings.
+ */
+
+const ModelClient = require("@azure-rest/ai-inference").default,
+  { isUnexpected } = require("@azure-rest/ai-inference");
+const { AzureKeyCredential } = require("@azure/core-auth");
+const { DefaultAzureCredential } = require("@azure/identity");
+const fs = require("fs");
+
+// Load the .env file if it exists
+require("dotenv").config();
+
+// You will need to set these environment variables or edit the following values
+const endpoint = process.env["ENDPOINT"] || "<endpoint>";
+const key = process.env["KEY"];
+const modelName = process.env["MODEL_NAME"];
+const imageFilePath = "sample1.png";
+const imageFormat = "png"; //"jpeg", "png", etc.
+
+async function main() {
+  console.log("== Image Embeddings Sample ==");
+
+  const client = createModelClient();
+
+  const image = getImageDataUrl(imageFilePath, imageFormat);
+
+  const response = await client.path("/images/embeddings").post({
+    body: {
+      input: [{ image }],
+      model: modelName,
+    },
+  });
+
+  if (isUnexpected(response)) {
+    throw response.body.error;
+  }
+  for (const data of response.body.data) {
+    console.log(data);
+  }
+  console.log(response.body.usage);
+}
+
+/*
+ * This function creates a model client.
+ */
+function createModelClient() {
+  // auth scope for AOAI resources is currently https://cognitiveservices.azure.com/.default
+  // auth scope for MaaS and MaaP is currently https://ml.azure.com
+  // (Do not use for Serverless API or Managed Computer Endpoints)
+  if (key) {
```

Comment:
I'm confused about the line `(Do not use for Serverless API or Managed Computer Endpoints)`. Where did that come from? I think it can be removed.

Reply:
removed